PEP8 in data

dipy/data/__init__.py:25:1: E402 module level import not at top of file
dipy/data/__init__.py:26:1: E402 module level import not at top of file
dipy/data/__init__.py:27:1: E402 module level import not at top of file
dipy/data/__init__.py:28:1: E402 module level import not at top of file
dipy/data/__init__.py:29:1: E402 module level import not at top of file
dipy/data/__init__.py:30:1: E402 module level import not at top of file
dipy/data/__init__.py:56:1: E402 module level import not at top of file
dipy/data/__init__.py:57:1: E402 module level import not at top of file
dipy/data/__init__.py:175:80: E501 line too long (89 > 79 characters)
dipy/data/__init__.py:204:80: E501 line too long (82 > 79 characters)
dipy/data/__init__.py:205:80: E501 line too long (120 > 79 characters)
dipy/data/fetcher.py:12:1: E402 module level import not at top of file
dipy/data/fetcher.py:13:1: E402 module level import not at top of file
dipy/data/fetcher.py:14:1: E402 module level import not at top of file
dipy/data/fetcher.py:16:1: E402 module level import not at top of file
dipy/data/fetcher.py:17:1: E402 module level import not at top of file
dipy/data/fetcher.py:19:1: E402 module level import not at top of file
dipy/data/fetcher.py:20:1: E402 module level import not at top of file
dipy/data/fetcher.py:21:1: E402 module level import not at top of file
dipy/data/fetcher.py:22:1: E402 module level import not at top of file
dipy/data/fetcher.py:283:80: E501 line too long (119 > 79 characters)
dipy/data/fetcher.py:300:80: E501 line too long (84 > 79 characters)
dipy/data/fetcher.py:310:8: E251 unexpected spaces around keyword / parameter equals
dipy/data/fetcher.py:310:10: E251 unexpected spaces around keyword / parameter equals
dipy/data/fetcher.py:321:80: E501 line too long (124 > 79 characters)
dipy/data/fetcher.py:635:1: E128 continuation line under-indented for visual indent
dipy/data/fetcher.py:698:80: E501 line too long (110 > 79 characters)
dipy/data/tests/test_fetcher.py:47:1: E302 expected 2 blank lines, found 1

--- a/dipy/data/__init__.py
+++ b/dipy/data/__init__.py
@@ -10,18 +10,6 @@ import json
 from nibabel import load
 from os.path import join as pjoin, dirname
 
-if sys.version_info[0] < 3:
-    import cPickle
-
-    def loads_compat(bytes):
-        return cPickle.loads(bytes)
-else:  # Python 3
-    import pickle
-    # Need to load pickles saved in Python 2
-
-    def loads_compat(bytes):
-        return pickle.loads(bytes, encoding='latin1')
-
 import gzip
 import numpy as np
 from dipy.core.gradients import GradientTable, gradient_table
@@ -55,6 +43,18 @@ from dipy.data.fetcher import (fetch_scil_b0,
 
 from ..utils.arrfuncs import as_native_array
 from dipy.tracking.streamline import relist_streamlines
+
+if sys.version_info[0] < 3:
+    import cPickle
+
+    def loads_compat(bytes):
+        return cPickle.loads(bytes)
+else:  # Python 3
+    import pickle
+    # Need to load pickles saved in Python 2
+
+    def loads_compat(bytes):
+        return pickle.loads(bytes, encoding='latin1')
 
 
 DATA_DIR = pjoin(dirname(__file__), 'files')
@@ -201,8 +201,10 @@ def get_data(name='small_64D'):
         'small_64D' small region of interest nifti,bvecs,bvals 64 directions
         'small_101D' small region of interest nifti,bvecs,bvals 101 directions
         'aniso_vox' volume with anisotropic voxel size as Nifti
-        'fornix' 300 tracks in Trackvis format (from Pittsburgh Brain Competition)
-        'gqi_vectors' the scanner wave vectors needed for a GQI acquisitions of 101 directions tested on Siemens 3T Trio
+        'fornix' 300 tracks in Trackvis format (from Pittsburgh
+            Brain Competition)
+        'gqi_vectors' the scanner wave vectors needed for a GQI acquisitions
+            of 101 directions tested on Siemens 3T Trio
         'small_25' small ROI (10x8x2) DTI data (b value 2000, 25 directions)
         'test_piesno' slice of N=8, K=14 diffusion data
         'reg_c' small 2D image used for validating registration

--- a/dipy/data/fetcher.py
+++ b/dipy/data/fetcher.py
@@ -4,11 +4,6 @@ import os
 import sys
 import contextlib
 
-if sys.version_info[0] < 3:
-    from urllib2 import urlopen
-else:
-    from urllib.request import urlopen
-
 from os.path import join as pjoin
 from hashlib import md5
 from shutil import copyfileobj
@@ -20,6 +15,12 @@ import tarfile
 import zipfile
 from dipy.core.gradients import gradient_table
 from dipy.io.gradients import read_bvals_bvecs
+
+if sys.version_info[0] < 3:
+    from urllib2 import urlopen
+else:
+    from urllib.request import urlopen
+
 
 # Set a user-writeable file-system location to put files:
 dipy_home = pjoin(os.path.expanduser('~'), '.dipy')
@@ -312,7 +313,8 @@ fetch_taiwan_ntu_dsi = _make_fetcher(
      'a95eb1be44748c20214dc7aa654f9e6b',
      '7fa1d5e272533e832cc7453eeba23f44'],
     doc="Download a DSI dataset with 203 gradient directions",
-    msg="See DSI203_license.txt for LICENSE. For the complete datasets please visit : http://dsi-studio.labsolver.org")
+    msg="See DSI203_license.txt for LICENSE. For the complete datasets please visit : \
+         http://dsi-studio.labsolver.org")
 
 fetch_syn_data = _make_fetcher(
     "fetch_syn_data",
@@ -337,7 +339,7 @@ fetch_mni_template = _make_fetcher(
     ['6e2168072e80aa4c0c20f1e6e52ec0c8',
      'f41f2e1516d880547fbf7d6a83884f0d',
      '1ea8f4f1e41bc17a94602e48141fdbc8'],
-    doc = "Fetch the MNI T2 and T1 template files")
+    doc="Fetch the MNI T2 and T1 template files")
 
 fetch_scil_b0 = _make_fetcher(
     "fetch_scil_b0",
@@ -346,7 +348,8 @@ fetch_scil_b0 = _make_fetcher(
     ['datasets_multi-site_all_companies.zip'],
     ['datasets_multi-site_all_companies.zip'],
     None,
-    doc="Download b=0 datasets from multiple MR systems (GE, Philips, Siemens) and different magnetic fields (1.5T and 3T)",
+    doc="Download b=0 datasets from multiple MR systems (GE, Philips, Siemens) \
+         and different magnetic fields (1.5T and 3T)",
     unzip=True)
 
 fetch_viz_icons = _make_fetcher("fetch_viz_icons",
@@ -657,8 +660,7 @@ def fetch_cenir_multib(with_raw=False):
                          '4e4324c676f5a97b3ded8bbb100bf6e5'])
 
     files = {}
-    baseurl = \
-'https://digital.lib.washington.edu/researchworks/bitstream/handle/1773/33311/'
+    baseurl = 'https://digital.lib.washington.edu/researchworks/bitstream/handle/1773/33311/'
 
     for f, m in zip(fname_list, md5_list):
         files[f] = (baseurl + f, m)

